content/participants/ttjc@KumaranSahi/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Muthu Kumaran
+type: "participant"
+name: "Muthu Kumaran"
+portfolio: "https://tanaypratap.com"
```

Comment:
Hey! @KumaranSahi Please add your portfolio link here and if you don't have one leave it blank.